discourse.Dockerfile

```diff
@@ -68,9 +58,10 @@ RUN ln -s /usr/share/zoneinfo/UTC /etc/localtime \
 # Run build process.
     && git -C "${CONTAINER_APP_ROOT}" clone --depth 1 --branch "${CONTAINER_APP_VERSION}" https://github.com/discourse/discourse.git app
 
-# Apply patch for LP#1903695
+# Apply patches
 COPY image/patches /srv/patches
 RUN git -C "${CONTAINER_APP_ROOT}/app" apply /srv/patches/lp1903695.patch \
+    && git -C "${CONTAINER_APP_ROOT}/app" apply /srv/patches/anonymize_user.patch \
```

Comment:
We should capture somewhere where/how we're trying to get this upstream, so that we can track if/when it lands and we can remove the custom patch.

Reply:
This won't land in this Discourse version. We'll have to upgrade.